### ISSUE TITLE
[BUG] Test execution time for each file is wrong in HTML report

### ISSUE THREAD

**Issue by Nav-2d** [microsoft/playwright#11334]
**Context:**
- Playwright Version: 1.17.2
- Operating System: Mac
- Node.js version: 12.22
- Browser: All

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```javascript
const { test, expect } = require('@playwright/test');

test('basic test', async ({ page }) => {
  await page.goto('https://playwright.dev/');
  const title = page.locator('.navbar__inner .navbar__title');
  await expect(title).toHaveText('Playwright');
});
```

```javascript
//config file
workers: 2,
reporter: [ ['list'], ['html'] ],
projects: [
  {
    name: 'Desktop Chrome',
    use: { browserName: 'chromium' },
  },
  {
    name: 'Desktop Safari',
    use: { browserName: 'webkit' },
  },
]
```

**Describe the bug**

Run the tests 10 times using `--repeact-each=10` and observe the total execution time displayed in terminal via list reporter (~20s) vs total execution time displayed in HTML reporter (~30s).

Currently, HTML report adds up the execution time for each test even though they are running in parallel.

![Screen Shot 2022-01-11 at 2 18 27 PM](https://user-images.githubusercontent.com/22205721/149030318-62989416-ab5c-45a1-b38f-23095855b3cb.png)

Add any other details about the problem here.

**Comment by ValorHeart** [comment]
Suffering from the same issue, it would be nice to have it fixed so the report users will know the actual execution time and how playwright parallel executions benefits

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your contribution to our project. This issue has been closed due to its limited upvotes and recent activity, and insufficient feedback for us to effectively act upon. Our priority is to focus on bugs that reflect higher user engagement and have actionable feedback, to ensure our bug database stays manageable.

Should you feel this closure was in error, please create a new issue and reference this one. We're open to revisiting it given increased support or additional clarity. Your understanding and cooperation are greatly appreciated.
